lib/models/builder.js

```diff
@@ -61,20 +80,36 @@ class Builder extends CoreObject {
       options = {
         tmpdir: tmpDir,
       };
-    } else {
```

Comment:
We still need this else case. We need to be able to disable the `BROCCOLI_2` experiment if there are significant issues.

Reply:
I disagree. The `return` here https://github.com/ember-cli/ember-cli/pull/8083/files#diff-53c8b5d6578c6f2fb2869c4d31bdc3a1R86 will ensure this code path is not un, unless BROCCOLI_2 experiment is disable OR `.read/.rebuild` error is thrown during brocccoli 2 builder construction.

Reply:
Awesome, thank you for pointing that out.